lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -2785,15 +2785,23 @@ ParseResult FIRStmtParser::parseAssert() {
       parseExp(predicate, "expected predicate in 'assert'") ||
       parseExp(enable, "expected enable in 'assert'") ||
       parseGetSpelling(message) ||
-      parseToken(FIRToken::string, "expected message in 'assert'") ||
-      parseToken(FIRToken::r_paren, "expected ')' in 'assert'") ||
-      parseOptionalName(name) || parseOptionalInfo())
+      parseToken(FIRToken::string, "expected message in 'assert'"))
```

Comment:
"message" is fine, especially since that's what we call it on the op, but maybe should be "format string"?